### PR TITLE
FIX isHidden for terra button with options

### PR DESCRIPTION
**Pull request by Dominik2809**
@plentymarkets/team-terra

#### Must be done by Terra

Testing
- [x] Person 1 (mandatory)
- [x] Person 2 (mandatory)

Browser-Support (relevant for changes in scss / appearance of components and jQuery)
- [x] Chrome
- [x] Firefox
- [x] Safari

----

#### Must be done by every developer
Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline). 

Documentation
- [x] Changelog (optional: relevant for every change which influences the API)

Testing
- [x] Unit Test (updated or created)

### CODE REVIEW

**Comment by dantrau** [comment]
@Dominik2809 please add a changelog entry

**Comment by nappynapster** [comment]
Bitte noch changelog hinzufügen

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<a name="4.7.0"></a>
+# 4.7.0 (unreleased)
+
+### Bug Fixes
+* **terra-button-with-options** `isHidden` option is now interpreted correcty.
```
Comment (dantrau) [inline CHANGELOG.md]:
```suggestion
* **terra-button-with-options** `isHidden` option is now interpreted correctly.
```

**Comment by maxhelligeplenty** [comment]
Hallo  ,
dantrau commented on this pull request.
> @@ -1,3 +1,9 @@
+
+# 4.7.0 (unreleased)
+
+### Bug Fixes
+* **terra-button-with-options** `isHidden` option is now interpreted correcty.
```suggestion
* **terra-button-with-options** `isHidden` option is now interpreted correctly.
```
-- You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1016#pullrequestreview-310550685
Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
nappynapster approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1016#pullrequestreview-310549991

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
dantrau approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1016#pullrequestreview-310550773

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
Merged #1016 into 4.X.X.

-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1016#event-2763726812

Test.test.test

**Comment by maxhelligeplenty** [comment]
Hallo  ,
TimWieder approved this pull request.





-- 
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub:
https://github.com/plentymarkets/terra-components/pull/1016#pullrequestreview-310551116

Test.test.test

**Pull request merged**